api/v1alpha1/ratelimitfilter_types.go

```diff
@@ -0,0 +1,133 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
```

Comment:
I don't the client stubs generated in this PR. Additionally, I don't think we need to generate them at this point (maybe for [egctl](https://github.com/envoyproxy/gateway/pull/797) in the future).

Reply:
thanks done